apps/betterangels-backend/clients/types.py

```diff
@@ -371,3 +371,10 @@ class ImportClientProfileInput:
     source_name: auto
     raw_data: auto
     client_profile: CreateClientProfileInput
+
+
+# Input for updating a client document
+@strawberry_django.input(Attachment, fields=["id", "original_filename"])
+class UpdateClientDocumentInput:
+    id: strawberry.ID
```

Comment:
`ID` is imported at the top of the file, so this can just be
```suggestion
    id: ID
```